<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_ListCustomers</class>
 <widget class="QFrame" name="F_ListCustomers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="L_header">
     <property name="text">
      <string>Manage Customers</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutHeader">
     <item>
      <layout class="QHBoxLayout" name="layoutRowBox">
       <item>
        <widget class="QLabel" name="L_show">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_rows">
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSpinBox" name="Sb_IDCustomer">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_editRow">
       <property name="text">
        <string>(*)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutSearchBox">
       <item>
        <widget class="QLabel" name="L_search">
         <property name="text">
          <string>Search : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_search"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layoutInputs">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="layoutCompany">
          <item>
           <widget class="QLabel" name="L_company">
            <property name="text">
             <string>Company</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_company"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutName">
          <item>
           <widget class="QLabel" name="L_name">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_name"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutEmail">
          <item>
           <widget class="QLabel" name="L_email">
            <property name="text">
             <string>Email Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_email"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutPhone">
          <item>
           <widget class="QLabel" name="L_phone">
            <property name="text">
             <string>Phone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_phone"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutPriceGroup">
          <item>
           <widget class="QLabel" name="L_priceGroup">
            <property name="text">
             <string>Price Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_priceGroup"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutCustomerGroup">
          <item>
           <widget class="QLabel" name="L_customerGroup">
            <property name="text">
             <string>Customer Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_customerGroup"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutDeposite">
          <item>
           <widget class="QLabel" name="L_deposite">
            <property name="text">
             <string>Deposite</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_deposite"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutNumber">
          <item>
           <widget class="QLabel" name="L_vatNumber">
            <property name="text">
             <string>VAT Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_vatNumber"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutAwdPoints">
          <item>
           <widget class="QLabel" name="L_awdPoints">
            <property name="text">
             <string>Awards Points</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_awdPoints"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutSubmit">
        <item>
         <widget class="QPushButton" name="Bt_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_cancel">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFooter">
     <item>
      <widget class="QLabel" name="L_info">
       <property name="text">
        <string>Showing 1 to 2 of 2 entries</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutPagesCTR">
       <item>
        <widget class="QPushButton" name="Bt_previous">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_pages"/>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_next">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
